pxt.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "music",
-    "description": "Sound and Music",
-    "version": "0.0.5",
+    "name": "makecode-minecraft-music",
```

Comment:
Is this what will appear in the extension card when they're adding it?

Reply:
Yes

Reply:
Hmm that's a little unfortunate. If you haven't already, probably worth filing a work item to have a display name field too, like Richard suggested in stand up.